README.md

```diff
@@ -1,39 +1,109 @@
-Automatically adds or removes labels from issues. You define the labels you'd like to add and/or remove in the YAML file. You can also specify if an issue should be ignored if an assignee has been added.
+Automatically adds or removes labels from issues, pull requests and project cards.
 
-To add it to your workflow:
+
+## Supported Github events
+- 'issues'
+- 'pull_request'
+- 'project_card'
+
+## repo-token
+To use this action a github access token is required since the github api is used. See the [oficial documentation](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) for more info.
+
+
+## add-labels & remove-labels
+to add or remove labels the parameters are:
+- `add-labels`
+- `remove-labels` 
+
+They may receive multiple labels separated by commas.
+They also may be used together. 
 
 ```yml
-    - uses: andymckay/labeler@1.0.2
-      with:
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        add-labels: "needs-triage, bug"
-        ignore-if-assigned: true
+add-labels: 'label1, label2'
+remove-labels: 'label1, label2'
 ```
 
-This adds the `needs-triage` and `bug` labels to the issue. The most common approach is to do this when issues are created, you can do this with the following in your workflow file:
+## Complete 'basic' usage
 
 ```yml
+name: issue-automation
+
 on:
   issues:
     types: [opened]
+  pull_request:
+    types: [opened]
+  project_card:
+    types: [moved]
+
+jobs:
+  automate-issues-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: initial labeling
+        uses: andymckay/labeler@1.0.2
```

Comment:
Could make this `@master` please. It goes against some of the guidance, but then we won't need to change this each time we do a release. I'd like to do one after this has merged in.